modules/admin_manual/pages/configuration/server/config_apps_sample_php_parameters.adoc

```diff
@@ -516,6 +516,28 @@ Please note, only one group can be defined. Default = empty = no restriction.
 'collabora_group' => '',
 ....
 
+== App: S3 Primary Object Storage
+
+Possible keys: `objectstore` ARRAY
+
+=== Configure the access parameters for a particular S3 provider.
+
+The detailed configuration of that array depends on the S3 provider.
+This example can therefore only show the general construct.
+See the "S3 Compatible Object Storage as Primary Storage Location" documentation for more details.
+
+==== Code Sample
+
+[source,php]
+....
+'objectstore' => [
+	'class' => 'Implementation\Of\OCP\Files\ObjectStore\IObjectStore',
```

Comment:
```suggestion
	'class' => 'OCA\Files_Primary_S3\S3Storage',
```

How about a real world string there? -- I see it as a fixed string. Is there a reason to consider it as a variable? The only other implementation I know of is objectstore -- which is deprecated.

Reply:
See my answer in core